lib/board.rb

```diff
@@ -27,13 +70,41 @@ def valid_coordinate?(coordinate)
 
   
   def valid_placement?(ship, coordinates)
-    coordinates.length == ship.length
-    cruiser_cons = cells.each_cons(3) {|cell| require 'pry'; binding.pry}
+    length_equals_ship(ship, coordinates) && consecutive_coordinates(ship, coordinates) 
   end
+    
+    
+
   
 
   #helper method 
-  def consecutive_coordinates
 
-  end
+    def length_equals_ship(ship, coordinates)
+      coordinates.length == ship.length
+    end
+
+    def consecutive_coordinates(ship, coordinates)
+      @valid_cruiser.include?(coordinates) || @valid_submarine.include?(coordinates)
+    end
+
+    
+    
+  # def consecutive_coordinates?
```

Comment:
come back, maybe try to make a dynamic method for selecting consecutive coordinates over the valid_cruiser an sub instance variables